Makefile

```
@@ -0,0 +1,42 @@
+TARGET_DIR?=$(shell pwd)
+PROTO_DIR=src/main/proto
+SRC_MAIN_PROTO_DIR=src/main/proto
+GITHUB_GROUP=github.com/th2-net
+
+TH2_GRPC_COMMON=th2-grpc-common
+TH2_GRPC_COMMON_URL=$(GITHUB_GROUP)/$(TH2_GRPC_COMMON)@makefile
```

I think we need to use the latest tag on master branch in th2-grpc-common repo
https://github.com/th2-net/th2-grpc-common/tree/v3.14.0
```suggestion
TH2_GRPC_COMMON_URL=$(GITHUB_GROUP)/$(TH2_GRPC_COMMON)@v3.14.0
```